src/components/ContentHighlights/data/hooks.js

```diff
@@ -68,6 +101,18 @@ export function useContentHighlightsContext() {
     }));
   }, [setState]);
 
+  const deleteSelectedRowId = useCallback((rowId) => {
+    const currentRowIds = { ...currentSelectedRowState };
+    delete currentRowIds[rowId];
+    setState(s => ({
```

Comment:
I believe we can nix the `currentSelectedRowState` state variable in favor of something like below:

```jsx
const deleteSelectedRowId = useCallback((rowId) => {
    const currentRowIds = { ...currentSelectedRowState };
    setState((s) => {
      const currentRowIds = { ...s };
      delete currentRowIds[rowId];
      return {
       ...s,
       stepperModal: {
         ...s.stepperModal,
         currentSelectedRowIds: currentRowIds,
       },
      };
    });
});
```